### PR TITLE
Changes to 2018 Open Challenge

### PR DESCRIPTION
**Pull request by justinhart**
As discussed by the TC in the March 23, 2018 meeting.

### CODE REVIEW

File: tests/OpenChallenge.tex [hunk 1]
```diff
@@ -15,7 +15,10 @@ \subsection{Task}
 \subsection{Presentation} 
 During the demonstration, the team can present the addressed problem and the demonstrated approach.
 \begin{itemize}
-\item A video projector or screen, if available, may be used to present a brief (max. 1 minute) introduction to what will be shown. 
+\item A video projector or screen, if available, may be used to present a brief (max. 2 minute) presentation relevant to the demonstration.
```
Comment (kyordhel) [inline tests/OpenChallenge.tex]: ...brief (max. 2 minute) **video** to introduce the demonstration.

File: tests/OpenChallenge.tex [hunk 2]
```diff
@@ -28,19 +31,15 @@ \subsection{Jury evaluation}
   \item \textbf{Jury of team leaders:} All teams have to provide \emph{one} person 
   (preferably the team-leader) to follow and evaluate the entire Open Challenge.
   \item \textbf{Evaluation:} Both the demonstration of the robot(s), and the answers of the team in the interview part are evaluated.\\ 
-  For each of the following \emph{evaluation criteria}, a maximum of \scoring{10 points} is given per jury member:
+  For each of the following \emph{evaluation criteria}, each jury member submits a score from $0-100$:
```
Comment (kyordhel) [inline tests/OpenChallenge.tex]: Why? I find easier to score 1-10.
I can live with that though...

File: tests/OpenChallenge.tex [hunk 3]
```diff
@@ -28,19 +31,15 @@ \subsection{Jury evaluation}
...
   \begin{enumerate}
-  \item Overall demonstration
```
Comment (kyordhel) [inline tests/OpenChallenge.tex]: I wouldn't remove this. There can be innovative and successful presentations carried out in a very boring way.

**Comment by balkce** [comment]
That is more or less what it was agreed to.

**Comment by justinhart** [comment]
This works for me, though, I think that we should put "Difficulty" into the
Open Challenge scoring criteria, explicitly, if we move to this sheet,
because that keyword was removed.

**Comment by kyordhel** [comment]
The changes including *difficulty* and *overall demo* are trivial and already done. I just need to push them up. If you all agree, I can do that and merge this.

**Comment by justinhart** [comment]
Well, I'm on board, but I can't approve it since it's my pull request.

**Comment by kyordhel** [comment]
Let's finish this.

**Pull request merged**